Update new-issue.md: update issue template (#5389)
xonsh/xonsh: .github/ISSUE_TEMPLATE/new-issue.md

---

> Hi! Thank you for xonsh! I faced with issue.

Reviewer: ```suggestion
```

Forcing an automated greeting doesn't really add anything except for extra text

## Current Behavior

> <!--- 
> Try to repeat your issue in `xonsh --no-rc --no-env`.
> Provide step by step repeatable description with commands and output.
> Use `$XONSH_SHOW_TRACEBACK=1`, `$XONSH_TRACE_SUBPROC=2` or `$XONSH_DEBUG=1`
> to have more info.
> -->

Reviewer: ```suggestion
<!--- 
For general xonsh issues, please try to replicate the failure using `xonsh --no-rc --no-env`.

Short, reproducible code snippets are highly appreciated.

You can use `$XONSH_SHOW_TRACEBACK=1`, `$XONSH_TRACE_SUBPROC=2`, or `$XONSH_DEBUG=1`
to collect more information about the failure
-->
```

Traceback (if applicable):

<details>

```xsh

> # Put here the long traceback.

Reviewer: ```suggestion
# Please paste the traceback here.
```

```

</details>

## Expected Behavior
<!--- What you expect -->

## xonfig

<details>

```xsh

> # Put here the output of `xonfig` command.

Reviewer: ```suggestion
# Please paste the output of the `xonfig` command here.
```
